scripts/openstack.py

```diff
@@ -224,13 +224,15 @@ def get_openstack_parser():
     )
     parser.add_argument(
         '--controller-ram',
-        help='override default minimum RAM (in megabytes) when selecting flavor for teuthology VM',
```

Comment:
All changes in this file are not related to the PR.

Reply:
The message in the commit does not correspond to changes. This does not look like cleanup.

Reply:
seems not major, do you want me to add it back with different commit message?